client/state/signup/steps/reducer.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import siteTitle from './site-title/reducer';
+
+export default combineReducers( {
+	siteTitle,
```

Comment:
A common pattern, for good reason: it allows you to write a simpler reducer (since the `siteTitle` reducer only cares about one thing, thus returning a simple string), and allows the `state.signup.steps` branch to be extended in the future by simply adding reducers here.